packages/react-ui-ag/src/Buttons/ApplyButton.js

```diff
@@ -0,0 +1,49 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import { Button } from '@rentpath/react-ui-core'
+
+@themed(/^ApplyButton/, {
+  pure: true,
+})
+
+export default class ApplyButton extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    handleClick: PropTypes.func,
+    theme: PropTypes.object,
+    name: PropTypes.node,
+  }
+
+  static defaultProps = {
+    theme: {},
+    name: 'Apply',
+  }
+
+  render() {
+    const {
+      theme,
+      name,
+      className,
+      handleClick,
+      ...props
+    } = this.props
+
+    if (!handleClick) return null
+
+    return (
+      <Button
+        className={classnames(
+          theme.ApplyButton,
+          className,
+        )}
+        onClick={handleClick}
+        data-tid="apply-button"
+        {...props}
```

Comment:
Any `onClick` passed will override your `handleClick` because of this.  Is that what you want?

Reply:
Probably not. I'll move it. Good call.